--- libSavitar-1ad7ddb202ec36f6b486b1f70279329ec0b8cc48/CMakeLists.txt~	2017-05-01 22:26:43.000000000 +0200
+++ libSavitar-1ad7ddb202ec36f6b486b1f70279329ec0b8cc48/CMakeLists.txt	2017-05-14 14:51:34.791994861 +0200
@@ -15,9 +15,9 @@
     find_package(SIP REQUIRED)
     if(EXISTS /etc/debian_version)
         # Running on a debian-based system, which requires special handling for python modules.
-        set(PYTHON_SITE_PACKAGES_DIR lib/python3/dist-packages CACHE STRING "Directory to install Python bindings to")
+	set(PYTHON_SITE_PACKAGES_DIR lib${LIB_SUFFIX}/python3/dist-packages CACHE STRING "Directory to install Python bindings to")
     else()
-        set(PYTHON_SITE_PACKAGES_DIR lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages CACHE STRING "Directory to install Python bindings to")
+        set(PYTHON_SITE_PACKAGES_DIR lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages CACHE STRING "Directory to install Python bindings to")
     endif()
     include_directories(python/ src/ ${SIP_INCLUDE_DIR} ${PYTHON_INCLUDE_DIR})
 endif()
@@ -56,7 +56,7 @@
 set(SAVITAR_VERSION 0.1.0)
 set(SAVITAR_SOVERSION 0)
 
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}")
 
 if(BUILD_STATIC)
     add_library(Savitar STATIC ${savitar_SRCS})
@@ -116,4 +116,4 @@
     ${CMAKE_BINARY_DIR}/SavitarConfig.cmake
     ${CMAKE_BINARY_DIR}/SavitarConfigVersion.cmake
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Savitar
-)
\ No newline at end of file
+)
