--- libSavitar-2.7.0/CMakeLists.txt~	2018-11-04 13:56:10.000000000 +0100
+++ libSavitar-2.7.0/CMakeLists.txt	2018-11-04 13:56:42.291239069 +0100
@@ -72,7 +72,7 @@
 if(BUILD_PYTHON)
     set(SIP_EXTRA_FILES_DEPEND python/Types.sip python/MeshData.sip python/SceneNode.sip python/Scene.sip)
     #set(SIP_EXTRA_SOURCE_FILES python/Types.cpp)
-    set(SIP_EXTRA_OPTIONS -g) # -g means always release the GIL before calling C++ methods.
+    set(SIP_EXTRA_OPTIONS -g -n PyQt5.sip) # -g means always release the GIL before calling C++ methods.
     add_sip_python_module(Savitar python/ThreeMFParser.sip Savitar)
 endif()
 
